articles/azure/azs-how-config-vm-update-management.md

```diff
@@ -0,0 +1,196 @@
+
+    - Select **Log Analytics Workspaces**.
+
+    - Click **Add** and in the *new* blade, select choices for the following items:
+
+        - A *Name* for the **Log Analytics workspace**. Example: `DefaultLAWorkspace`.
+
+        - A *Subscription* to link the workspace to.
+
+        - A *Resource Group* to link the workspace to.
+
+        - A *Location* available to host the workspace on.
+
+        > [!NOTE]
+        > The currently supported locations are: *West Central US*, *East US*, *Canada Central*, *UK South*, *West Europe* and *Southeast Asia*.
+
+        - *Pricing Tier* to use.
+
+        Here is an example of the blade and all items filled out:
+
+        ![Log Analytics workspace creation](images/azs-browser-example-log-analytics-workspace.PNG)
+
+    - Press **OK** once finished. The **Log Analytics workspace** will now begin deploying. A notification will appear on the top right of the portal.
+
+    - Once deployment is complete, navigate to the *resource group* you placed the **Log Analytics workspace** within.
+
+    - Click the newly create workspace. On the *new* blade, under settings, select *Advanced settings*, then select *Windows* or *Linux* servers depending on the *VM Type* you wish to add analytics too. 
+    Note down the **Workspace ID** and **Primary Key** values.
+
+        ![Log Analytics workspace advanced settings](images\azs-browser-log-analytics-workspace-advanced-settings.PNG)
+
+    - Within *Advanced settings*, Select the **Data** blade, and click *`VM Type` Performance Counters.*
+
+    - Ensure all *counters* are selected and click **Add the selected performance counters**.
+
+    - Click **Save**.
+
+        ![Performance Counters](images/azs-browser-example-performance-counters.PNG)
+
+        > [!NOTE]
+        > For Linux Performance Counters, ensure **Apply below configuration to my machines** is selected.
+
+3. Create and configure an *Automation Account*:
+
+    - Click *+ Create a resource*.
+
+    - Search `automation` and click **Create**.
+
+        ![automation account](images\azs-browser-search-automation-account.PNG)
+
+    - Choose a *Name*, *Subscription*, *Resource Group*, *Location* and ensure *Create Azure Run As Account* is set to **Yes**.
+
+    - Click **Create**.
+
+        ![Add automation account](images/azs-browser-add-automation-account.PNG)
+
+        > [!NOTE]
+        > The error "Azure Classic Run As account creation error" may occur. This **DOES NOT** effect the process.
+
+    - Once deployed, navigate to the *Automation Account*. In the *new* blade, Under *Configuration Management*, select the solution to enable (Inventory & Change Tracking or Update management).
+
+    - Select the *Log Analytics workspace* to link the automation account too.
+
+    - Click **Enable**.
+
+    - Wait for the deployment to complete.
+
+    - Repeat the two previous steps for all two solutions (Inventory & Change Tracking and Update management).
```

Comment:
all two doesn't sound right. Maybe:
Repeat the previous two steps for both Inventory & Change Tracking and Update management.
Also, I think you should make it very specific which two steps you're asking users to repeat. I'm assuming you mean the numbered steps 2 and 3 rather than the bullet points, so:
Repeat steps 2 and 3 for both Inventory & Change Tracking and Update management.
It's also probably worth having this as a numbered steps on its own rather than hidden at the end of a list of sub-steps. So this should become step 4